### DISCUSSION TITLE
Error in GitHub Actions when Checking R Package with rhub:

### DISCUSSION THREAD

**Discussion by SYOUNG9836** [r-hub/rhub#622] [Q&A] [answered]
I'm currently developing my own R package and preparing it for submission to CRAN. To ensure everything is set up correctly, I'm using GitHub Actions with the rhub package to check my package. However, I've encountered an error that seems to occur in the YAML file during the GitHub Actions workflow.

Here's what I've added to my DESCRIPTION file for handling dependencies hosted on a drat repository:

Remotes: kadmin = url::~~~~kadmin_1.0.0.tar.gz
Additional_repositories: url::~~~~drat

The error appears to be linked to how the GitHub Actions YAML is configured or possibly how the dependencies are specified in the DESCRIPTION file. Could anyone provide guidance on how to correctly configure the YAML file? Any help would be greatly appreciated!

rhub.yaml link

![image](https://github.com/r-hub/rhub/assets/78527477/ca79e548-12c2-4ab0-869e-5f55f603c482)
![image](https://github.com/r-hub/rhub/assets/78527477/098f0920-9236-4fcb-9108-44030bc75d82)

**Comment by gaborcsardi** [comment, marked as answer]
First, remove the spaces from the `Remotes` field.

**Reply by SYOUNG9836** [reply]
I keep encountering errors.

![image](https://github.com/r-hub/rhub/assets/78527477/33c4ed40-57c8-4f41-83a6-044940dcba21)

**Reply by gaborcsardi** [reply]
Can you share your package? OR at least your `DESCRIPTION` file?

**Reply by SYOUNG9836** [reply]
Here's the link to the DESCRIPTION file. I've also made the package public.

https://github.com/SYOUNG9836/rNFI/blob/main/DESCRIPTION

**Reply by SYOUNG9836** [reply]
Thank you so much for your assistance with my problem so far. It turns out I had some incorrect code in the DESCRIPTION file. I fixed it, and that solved the problem. I changed the address to: Remotes: kadmin=github::SYOUNG9836/kadmin"

**Reply by SYOUNG9836** [reply]
![image](https://github.com/r-hub/rhub/assets/78527477/2a63a6d3-8dd7-48b9-aa39-27c9f1d9207d)

I apologize. I have one more question.

Error related to tcltk:
no DISPLAY variable so Tk is not available

Could this issue also be due to the r code rather than the yaml file?"

**Reply by gaborcsardi** [reply]
This seems like a separate issue, can you start another thread, please? Also, on what OS does this happen?
